cmd/qemu/src/lib.rs

```diff
@@ -117,12 +145,52 @@ fn qemu(context: &mut humility::ExecutionContext) -> Result<()> {
     }
 
     // Ignore ctrl c so qemu and or gdb can handle it
-    if subargs.gdb {
+    if subargs.gdb || subargs.command.is_some() {
         // start qemu in the background
         cmd.stdin(Stdio::piped());
         let _qemu = Runner(cmd.spawn().context("Could not start 'qemu'")?);
-        // now start gdb
-        gdb(context)?;
+        if subargs.gdb {
+            // now start gdb
+            gdb(context)?;
+        } else if let Some(command) = subargs.command {
+            // we unfornunatly have to contruct a new command from scratch, calling back into the
+            // base humility command parsers would create a circular dependency
+            let mut cmd = Command::new("humility");
```

Comment:
This assumes 'humility' is available on 'PATH'.  This may not necessarily be the case (or the version on humility currently executing may not be the same as the one found on the path. Ideally, we should try and make sure we invoke the same humility as is currently running.

Reply:
Looks like `std::env::current_exe()` can provide the path of the current-running executable: https://doc.rust-lang.org/std/env/fn.current_exe.html.

Reply:
Fixed.